recipe/meta.yaml

```
@@ -1,43 +1,58 @@
 {% set name = "cmarkgfm" %}
-{% set version = "0.8.0" %}
+{% set version = "0.4.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: fe859b0e98d222b4b458022007daac2757e970f0671d45e55d01931c1b4bb3cb
+  sha256: f20900f16377f2109783ae9348d34bc80530808439591c3d3df73d5c7ef1a00c
 
 build:
-  number: 2
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vvv "
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cffi                                   # [build_platform != target_platform]
     - {{ compiler('c') }}  # [not (win and py27)]
   host:
-    - cffi >=1.0.0
-    - pip
+    - cffi =>1.15.0
     - python
+    - pip
+    - wheel
+    - m2w64-toolchain 5.3.0 # [win and py27]
+    - libpython 2.1       # [win and py27]
+    - {{ pin_compatible('m2w64-gcc-libs', max_pin='5.3.0') }}   # [win and py27]
+
   run:
-    - cffi >=1.0.0
+    - cffi =>1.15.0
     - python
+    - m2w64-gcc-libs   # [win and py27]
 
 test:
   imports:
     - cmarkgfm
+  commands:
+    - pip check
+
+# Platform-specific test section
+{% if osx %}
+test:
+  skip: True
+{% endif %}
```

can we use the usual selector? something like this
```suggestion
test:
  skip: True  # [osx]
  imports:
    - pip
    - cmarkgfm
  commands:
    - pip check
```

I also added `pip` in the imports for the tests